src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -173,11 +173,24 @@ public Task GetAll()
 
             if (string.IsNullOrWhiteSpace(friendlyName))
                 throw new ArgumentException("'Name' cannot be empty when generating a new certificate");
+            
+            certificateJson.TryGet("Password", out password); //can be null
+
+            ValidatePermissions(certificateJson, out permissions, out serverAdmin);
+        }
+
+        private static void ValidatePermissions(BlittableJsonReaderObject certificateJson, out BlittableJsonReaderArray permissions, out bool serverAdmin)
+        {
+            certificateJson.TryGet("ServerAdmin", out serverAdmin); //can be null, default is false
 
             if (certificateJson.TryGet("Permissions", out permissions) == false)
                 throw new ArgumentException("'Permissions' is a required field when generating a new certificate");
-            
+
             const string validDbNameChars = @"([A-Za-z0-9_\-\.]+)";
+
+            if (permissions.Items.Count() > 0 && serverAdmin == true)
```

Comment:
Let us remove this logic. It is fine to add a server admin to a cert, admin may do that while troubleshooting, but not want to lose the specific dbs they added.